src/status_im/chat/models/loading.cljs

```diff
@@ -38,6 +38,7 @@
               (filters/load-filters))))
 
 (fx/defn offload-all-messages
+  {:events [::offload-all-messages]}
```

Comment:
how is this related to PR?

Reply:
This was called during `screens/on-will-focus` because this event is not called anymore I moved this into a separate event. It is called during the focus on home view.